tornado_server.py

```diff
@@ -75,6 +78,20 @@ def check_origin(self,origin):
 ])
 
 if __name__ == "__main__":
-    http_server = tornado.httpserver.HTTPServer(application)
+ 
+    # This line will connect to the website, read its contents
+    # and parse the JSON output
+	
+    data = loads(urlopen("http://httpbin.org/ip").read())
```

Comment:
Get the public IP address of the instance. If it matches our benedictation.io public IP, then we create a secure https Python server. Otherwise we create a regular insecure http Python server.